### PR TITLE
Feat/roman numerals

### PR DESCRIPTION
**Pull request by JarbasAl**
port https://github.com/HelloChatterbox/lingua-nostra/pull/28

## Summary by CodeRabbit

- **New Features**
  - Added the ability to convert Roman numerals to integers.
  - Introduced functionality to extract and normalize Roman numerals in English text.

- **Enhancements**
  - Improved text normalization by adding `normalize_roman_numerals` and `get_color` functions.

- **Dependencies**
  - Minor update to the `quebra_frases` package version specification (no functional change).

### CODE REVIEW

File: test/test_parse_common.py [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+class TestParseCommon(unittest.TestCase):
+    def setUp(cls) -> None:
+        load_language("en")
+
+    def test_roman(self):
+        # valid numerals
+        self.assertEqual(roman_to_int("III"), 3)
+        self.assertEqual(roman_to_int("IV"), 4)
+        self.assertEqual(roman_to_int("V"), 5)
+        self.assertEqual(is_roman_numeral("IV"), True)
```
Comment (NeonDaniel) [inline test/test_parse_common.py:31]: I'd add one more complex case here, i.e. `MCMLXXIIV == 1973`

**Review by NeonDaniel — CHANGES_REQUESTED** [review]

File: lingua_franca/parse.py [hunk 2]
```diff
@@ -15,15 +15,15 @@
 #
 import json
 from lingua_franca.util import match_one, fuzzy_match, MatchStrategy
-from lingua_franca.lang.parse_common import match_yes_or_no
+from lingua_franca.lang.parse_common import match_yes_or_no, extract_roman_numeral_spans, is_roman_numeral
```
Comment (NeonDaniel) [inline lingua_franca/parse.py:18]: should include `roman_to_int, is_roman_numeral, normalize_roman_numerals` for unit tests below

**Review by NeonDaniel — CHANGES_REQUESTED** [review]

**Comment by JarbasAl** [comment]
@coderabbitai review

File: test/unittests/test_parse_common.py [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+        self.assertEqual(roman_to_int("V"), 5)
+        self.assertEqual(roman_to_int("MCMLXXIV"), 1974)
+        self.assertEqual(roman_to_int("MCMLXXV"), 1975)
+        self.assertEqual(is_roman_numeral("IV"), True)
+
+        # invalid numerals
+        self.assertEqual(roman_to_int("v"), None)
+        self.assertEqual(is_roman_numeral("ii"), False)
+        self.assertEqual(is_roman_numeral("the IV century"), False)
+
```
[the remark this answers is not in the document]
Comment (JarbasAl) [reply test/unittests/test_parse_common.py:39]: yes

**Pull request closed without merging**